charts/tidb-operator/templates/admission/jobPatch/post-del-secret-job.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and (eq .Values.admissionWebhook.enabled true) (eq .Values.admissionWebhook.certProvider "patch") }}
```

Comment:
`certProvider: patch` sounds weird, how can `patch` be a provider?

Reply:
Using `autoGenerator` instead